src/components/Cart/CartCard/CartProduct.js

```diff
@@ -3,6 +3,7 @@ import { Delete, Negative, Positive } from "../../../assets";
 import { useData } from "../../../Context/stateManage/state-context";
```

Comment:
Suggestion You could show total quantity of items instead of cart items.
![image](https://user-images.githubusercontent.com/91944618/160815366-df26de61-c678-4f1f-b473-345185649e6b.png)